addons/point_of_sale/models/pos_order.py

```diff
@@ -595,6 +620,9 @@ def _default_company_id(self):
         readonly=True,
         states={'draft': [('readonly', False)]},
     )
+    pos_payment_ids = fields.One2many(comodel_name='pos.payment', inverse_name='pos_order_id', string='Pos Payments', readonly=True)
+    session_state = fields.Selection(string="Session State", related='session_id.state', readonly=True)
+    session_move_id = fields.Many2one('account.move', string='Session Journal Entry', related='session_id.move_id', readonly=True, copy=False)
```

Comment:
does not seems to be used, and copy false on not stored related porbably do nothing.